.gitlab-ci.yml

```diff
@@ -27,10 +27,19 @@ build-package:
 s3-release:
   image: $IMAGE_AWS_CI
   variables:
-    BUCKET: mattermost-developer-documentation/branches
+    BUCKET: mattermost-developer-documentation
   stage: deploy
   script:
-    - aws s3 cp dist/html s3://mattermost-developer-documentation/branches/master --recursive --cache-control no-cache --acl public-read
+  - |
+    echo "Remove all previous artifacts"
+    aws s3 rm s3://${BUCKET}/branches/master --recursive
+  - |
+    echo "Save new artifacts"
+    aws s3 cp dist/html s3://${BUCKET}/branches/master --recursive --cache-control no-cache --acl public-read
+  - |
```

Comment:
We can utilise `aws s3 sync` for more performance with the `--delete` option . 
```bash
aws s3 sync dist/html s3://mattermost-developer-documentation/branches/master --delete --cache-control no-cache --acl public-read --no-progress
```

More info [here](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/s3/sync.html)

Reply:
Oh nice, that works. I didn't realize the `--delete` option there.